### PR TITLE
Feature/add colors

### PR DESCRIPTION
**Pull request by azriel91**
Adds the ability to render log messages with colors.

The styling code is largely from [limira-rs/wasm-logger](https://gitlab.com/limira-rs/wasm-logger), though I cleaned it up a bit. I've put it behind a `"color"` feature flag and it is off by default.

What it looks like:

log_messages_styled

---

Between `console_log` and `wasm-logger`, I decided to submit the PR here since this already integrates with `fern`.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -67,26 +80,35 @@
...
 use web_sys::console;
 
+#[cfg(feature = "color")]
+use wasm_bindgen::JsValue;
+
+#[cfg(feature = "color")]
+const STYLE: style::Style<'static> = style::Style::default();
+
+#[cfg(feature = "color")]
+mod style;
```
Comment (iamcodemaker) [inline src/lib.rs:111]: Add `#[doc(hidden)]` here. The style stuff isn't relevant to external users.

File: src/style.rs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/// Log message styling.
+///
+/// Adapted from <https://gitlab.com/limira-rs/wasm-logger/-/blob/0c16227/src/lib.rs#L72-85>
+pub struct Style<'s> {
```
Comment (iamcodemaker) [inline src/style.rs]: If I'm not mistaken, this can be `pub(crate)`.

**Review by iamcodemaker — CHANGES_REQUESTED** [review]
Thanks for submitting a PR. Looks good. Just a few documentation/visibility related changes and this should be good to go.

**Comment by azriel91** [comment]
Oh yeaps, good points! (eagle eye for that detail)

**Review by iamcodemaker — APPROVED** [review]

**Comment by iamcodemaker** [comment]
Version `0.2.0` has been published with this change.

**Pull request merged**